doc/man3/flux_msg_create.rst

```diff
@@ -0,0 +1,79 @@
+==========================
+flux_msg_create(3)
+==========================
+
+.. default-domain:: c
+
+SYNOPSIS
+========
+
+.. code-block:: c
+
+   #include <flux/core.h>
+
+   flux_msg_t *flux_msg_create (int type)
+
+   flux_msg_t *flux_msg_copy (const flux_msg_t *msg, bool payload)
+
+   const flux_msg_t *flux_msg_incref (const flux_msg_t *const_msg)
+
+   void flux_msg_decref (const flux_msg_t *const_msg)
+
+   void flux_msg_destroy (flux_msg_t *msg)
+
+
+DESCRIPTION
+===========
+
+:func:`flux_msg_create` creates a :type:`flux_msg_t` of the :var:`type`.
+Different types of Flux messages are defined in RFC :doc:`rfc:spec_3`. All
+messages have a starting reference count of 1.
+
+:func:`flux_msg_copy` duplicates :var:`msg`. The payload is omitted unless
+:var:`payload` is true. The initial reference count of the new message is 1.
+
+:func:`flux_msg_incref` increments the reference count of :var:`const_msg`
+by 1.
+
+:func:`flux_msg_decref` decrements the reference count of :var:`const_msg`
+by 1. When the reference count reaches 0, the message is destroyed.
+
+:func:`flux_msg_destroy` destroys :var:`msg`.
+
+RETURN VALUE
+============
+
+:func:`flux_msg_create` returns a :type:`flux_msg_t` type. On failure,
+NULL is returned and EINVAL or ENOMEM is raised.
+
+:func:`flux_msg_copy` returns a new :type:`flux_msg_t` object with the same
+attributes as another :type:`flux_msg_t` objects. On failure, NULL is
+returned and EINVAL is raised because :var:`msg` is invalid.
```

Comment:
This should be trimmed down the same way.  Actually it could just be lumped in with flux_msg_create() in this section.